<launch>
    <node name="move_goal" pkg="cv_basics" type="move_goal.py" output="screen">
        <rosparam param="p_seq">[2, 2, 0]</rosparam>
        <rosparam param="yea_seq">[180]</rosparam>
    </node>

    <node name="second_goal" pkg="cv_basics" type="second_goal.py" output="screen" required="true">
        <rosparam param="p_seq2">[0, 0, 0]</rosparam>
        <rosparam param="yea_seq2">[0]</rosparam>
    </node>
</launch>